src/perf.ts

```diff
@@ -138,7 +138,12 @@ async function launchDesktop(options: Options, perfFile: string, markers: string
 		EXTENSIONS_FOLDER,
 		'--disable-extensions',
 		'--disable-workspace-trust',
-		'--disable-features=CalculateNativeWinOcclusion', // disable penalty for occluded windows
```

Comment:
This is redundant https://github.com/microsoft/vscode/blob/846afa279ab4a874c63d9c89b160772773463b44/src/main.ts#L317-L318